packages/react-icons/README.md

```diff
@@ -21,21 +21,41 @@ In order to use these icons, simply import them as `import { [Componentname][siz
 ```tsx
 import { AccessTime24Filled } from "@fluentui/react-icons";
 ```
-You can also style the icons using the `IFluentIconsProps`interface, with the `className` prop or the `primaryFill` prop
+You can also style the icons using the `IFluentIconsProps`interface, with the `className` prop or the `primaryFill` prop.
+
+Finally, you can bundle the `Filled` and `Regular` versions of each icon using the `bundleIcon` method. The `bundleIcon` function returns a component with both states of the icon, and provides default styling for `hover`, `active`, and `focus` states. You can then use the same props as the regular unbundled icons.
+
+If you would like to get the bundled icon without any of the default styling, then add the `filled` prop to the new component and the icon will not have any default styling. To style this new icon, simply add `.root-span.filled` and `.root-span.regular` to your user-defined stylesheet and you will be able to surface which icon you want.
+
+
+```scss
+.test-span {
+    .root-span.filled {
+        display: none;
+    }
+    .root-span.regular {
+        display: inline-block;
+    }
+}
+```
 
 ```tsx
 import * as React from "react";
 import * as ReactDOM from "react-dom";
-import { AccessTime24Filled } from "@fluentui/react-icons";
+import { AccessTime24Filled, AccessTime24Regular, bundleIcon } from "@fluentui/react-icons";
+import './Test.scss';
 
 const iconStyleProps: IFluentIconsProps = {
     primaryFill: "purple",
     className: "iconClass"
 };
+const BundledIcon = bundleIcon(AccessTime24Filled, AccessTime24Regular)
```

Comment:
This is super nitpicky, but in a 'real world' scenario this should be called AccessTime, not BundledIcon, I would think.

Reply:
Its been changed!